indy_node/test/nym_txn/test_nym_blacklisting.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from indy import did
+
+from indy_common.constants import TRUST_ANCHOR_STRING
+from plenum.common.constants import TRUSTEE_STRING, STEWARD_STRING
+from plenum.common.exceptions import RequestRejectedException
+from plenum.test.helper import sdk_get_and_check_replies, sdk_sign_and_submit_op
+from plenum.test.pool_transactions.helper import sdk_add_new_nym
+
+
+def test_steward_suspension_by_another_trustee(looper,
+                                               sdk_pool_handle,
+                                               sdk_wallet_trustee,
+                                               sdk_wallet_handle,
+                                               with_verkey):
+    new_trustee_did, new_trustee_verkey = looper.loop.run_until_complete(
+        did.create_and_store_my_did(sdk_wallet_trustee[0], "{}"))
+    new_steward_did, new_steward_verkey = looper.loop.run_until_complete(
+        did.create_and_store_my_did(sdk_wallet_trustee[0], "{}"))
+    """Adding new steward"""
+    sdk_add_new_nym(looper, sdk_pool_handle,
+                    sdk_wallet_trustee, 'newSteward', STEWARD_STRING, verkey=new_steward_verkey, dest=new_steward_did)
+    """Adding new trustee"""
+    sdk_add_new_nym(looper, sdk_pool_handle,
+                    sdk_wallet_trustee, 'newTrustee', TRUSTEE_STRING, verkey=new_trustee_verkey, dest=new_trustee_did)
+    op = {'type': '1',
+          'dest': new_steward_did,
+          'role': None}
+    if with_verkey:
+        op['verkey'] = new_steward_verkey
+    """Blacklisting new steward by new trustee"""
```

Comment:
I might be wrong, but it seems like this comment belongs before op assignment, not after